broker/tasks/sns.py

```diff
@@ -0,0 +1,71 @@
+import logging
+
+from botocore.exceptions import ClientError
+from sqlalchemy.orm.attributes import flag_modified
+
+from broker.aws import sns_commercial
+
+from broker.extensions import config, db
+
+from broker.models import Operation
+from broker.tasks import huey
+
+logger = logging.getLogger(__name__)
+
+
+@huey.retriable_task
+def create_notification_topic(operation_id: int, **kwargs):
+    operation = db.session.get(Operation, operation_id)
+    service_instance = operation.service_instance
+
+    operation.step_description = "Creating SNS notification topic"
+    flag_modified(operation, "step_description")
+    db.session.add(operation)
+    db.session.commit()
```

Comment:
(not for this PR)
this pattern is repeated in almost every task, should we refactor it out?